File: Dante1/Module1.02/Zadanie2.05/main.c
#include <stdio.h>

int main()
{
    int number;
    printf("Enter number: ");

    int check = scanf("%i",&number);
    if(!check)
    {
        printf("Incorrect input");
        return 1;
    }

    if(!number)
    {
        printf("Equal to zero");
    }
    else
    {
        printf("Not equal to zero");
    }
    return 0;
}

File: Dante1/Module1.02/Zadanie2.05/main.rs
fn main()
{
    println!("Enter: ");
    
    let mut input: String = String::new();
    std::io::stdin().read_line(&mut input).expect("Incorrect input");
    
    let input: i32 = input.trim().parse().expect("Incorrect number");
    
    if !input.is_negative() && !input.is_positive()
    {
        println!("Equal to zero");
    }
    else 
    { 
        println!("Not equal to zero");    
    }
    
    return;
}
